ddtrace/ext/ci.py

```diff
@@ -109,6 +109,11 @@ def tags(env=None, cwd=None):
     # is None or "" should be overwritten.
     tags.update({k: v for k, v in git_info.items() if not tags.get(k)})
 
+    user_specified_git_info = git.extract_user_git_metadata(env)
+
+    # Tags provided by the user take precedence over everything
+    tags.update({k: v for k, v in user_specified_git_info.items() if user_specified_git_info.get(k)})
```

Comment:
```suggestion
    tags.update({k: v for k, v in user_specified_git_info.items() if v})
```

This should be the same no?

Reply:
yes, good catch